### PR TITLE
Coldcard support (filepicker)

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/41n98pkdw6kwk7z92603cyaph0

[download apk](http://filestorage.bluewallet.io:1488/1577605976.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/bq527agc57rmv67a5nvryn4jn4

[download apk](http://filestorage.bluewallet.io:1488/1577646869.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/v4mttbhfxqdrzb0neufyq4tr78

[download apk](http://filestorage.bluewallet.io:1488/1577669954.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/79by1z3njpm4ybzuak07u1299g

[download apk](http://filestorage.bluewallet.io:1488/1577684587.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/2z0gjbqavbudrrva20m7x51qzr

[download apk](http://filestorage.bluewallet.io:1488/1577691946.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ptc3x55fdw1dd90hgfr9hg6j6w

[download apk](http://filestorage.bluewallet.io:1488/1577692747.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/c55ge783qc9wbbzcma9rncdqh4

[download apk](http://filestorage.bluewallet.io:1488/1577693369.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/x6jjawkb6uxre7uyjzuwkha4t0

[download apk](http://filestorage.bluewallet.io:1488/1577720805.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/f4qh6bmp5xqgca5cynfutgzhgm

[download apk](http://filestorage.bluewallet.io:1488/1577726666.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/h6mrt7prp913y1nk3xxydjvdjr

[download apk](http://filestorage.bluewallet.io:1488/1577734995.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/4qvtdu2vz2ewud352m8axx3mtm

[download apk](http://filestorage.bluewallet.io:1488/1577765997.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/6e7avfnqxz3menvtvy0rxp9qu4

[download apk](http://filestorage.bluewallet.io:1488/1577766430.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/zbjkgjb9e7b9jrufguaj6mb6jw

[download apk](http://filestorage.bluewallet.io:1488/1577770405.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/fwpgevynad8et4ye9zke0d1gm8

[download apk](http://filestorage.bluewallet.io:1488/1577777593.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/zewqewtaz0yygtwgw5f8ywtdac

[download apk](http://filestorage.bluewallet.io:1488/1577806722.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/8h6pynpq7c44pxh03bdjfx4vn0

[download apk](http://filestorage.bluewallet.io:1488/1577848456.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/41n3tkkwxv54vwnrypterf3wkm

[download apk](http://filestorage.bluewallet.io:1488/1577850079.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/8p7d2u7epp9ua9j1u7m6rr25pg

[download apk](http://filestorage.bluewallet.io:1488/1577855209.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/rtjp976yq0euvbtj3w7czzh3e8

[download apk](http://filestorage.bluewallet.io:1488/1577856389.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/3arpqgp953nm4k0zh5jwkk680m

[download apk](http://filestorage.bluewallet.io:1488/1577866613.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/153yxnv4vkvck7xb96crnm0kkg

[download apk](http://filestorage.bluewallet.io:1488/1577904309.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/qbgj50g26aaq55ugzaf4q3mygg

[download apk](http://filestorage.bluewallet.io:1488/1577905199.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/jqna3qc1xph9defqawkt3enftw

[download apk](http://filestorage.bluewallet.io:1488/1577934757.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/db0xcr25ymc9qw2555062egn0g

[download apk](http://filestorage.bluewallet.io:1488/1577942045.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/zdetw9t8mfh6xvv18vmct7ur98

[download apk](http://filestorage.bluewallet.io:1488/1577952843.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/vk2ytxt4aj8847crhbw3xam9j4

[download apk](http://filestorage.bluewallet.io:1488/1577977932.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/wzy0vdhbknthw70xdwbt8zr5fc

[download apk](http://filestorage.bluewallet.io:1488/1577984514.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/xfx3vvp8tnny41472gzzb0tnxr

[download apk](http://filestorage.bluewallet.io:1488/1577994623.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/7dd72q8027qxwe04xf7gajt7r4

[download apk](http://filestorage.bluewallet.io:1488/1577995714.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/9dph8ug8fyprq4wnm9q90wwfkg

[download apk](http://filestorage.bluewallet.io:1488/1577998956.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/vyrhfypyp3twqdmezat1tyh994

[download apk](http://filestorage.bluewallet.io:1488/1578001054.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/zyn07dj8hztjfaj95cwj7bgr3w

[download apk](http://filestorage.bluewallet.io:1488/1578003242.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/dvk62hujgtypy9crnm3tpxyd8w

[download apk](http://filestorage.bluewallet.io:1488/1578024657.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/amvwmwt52r3hm29tztxkjj5hvw

[download apk](http://filestorage.bluewallet.io:1488/1578108017.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/4vp502gzq5ubd4hg3def1t9a58

[download apk](http://filestorage.bluewallet.io:1488/1578115003.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/uxdmurq3gxrte8wxkmcugxg9gr

[download apk](http://filestorage.bluewallet.io:1488/1578120333.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/20vc80f5fx80bxgabc82j10gkr

[download apk](http://filestorage.bluewallet.io:1488/1578152670.apk)

**Comment by Overtorment** [comment]
image

**Comment by Overtorment** [comment]
it-юмор-geek-3596647

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/bjx74j34n0hdax0vry0820gyqw

[download apk](http://filestorage.bluewallet.io:1488/1578180288.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/zn1c91epvpwpfkqc4urjtkutjc

[download apk](http://filestorage.bluewallet.io:1488/1578186599.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/mvx31m95kyw4ww709hwuzzagzw

[download apk](http://filestorage.bluewallet.io:1488/1578306571.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/9whf2ag2v2x6yv05cdy3e9wpzr

[download apk](http://filestorage.bluewallet.io:1488/1578307171.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/5159b5acqrbyqx55xe6br12xkg

[download apk](http://filestorage.bluewallet.io:1488/1578380518.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/tmkm6un4b8ezvuaba0hzexe11g

[download apk](http://filestorage.bluewallet.io:1488/1578474989.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/jmgkcp0eacp3dc1ctqzkqj48d8

[download apk](http://filestorage.bluewallet.io:1488/1578511544.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/68nb93vhj7ff9wybt8vhd92654

[download apk](http://filestorage.bluewallet.io:1488/1579052516.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/qdcvb423px3k2wxb4hgh19nhyg

[download apk](http://filestorage.bluewallet.io:1488/1579064230.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/wz8wfjp2cq531dcaf2gkyxd5cg

[download apk](http://filestorage.bluewallet.io:1488/1579148250.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/pj381bwt5f4vxv9nek390f0h4r

[download apk](http://filestorage.bluewallet.io:1488/1579492365.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/phf3mwk3854jmqcuzdjfcd1hbw

[download apk](http://filestorage.bluewallet.io:1488/1579753223.apk)

**Comment by Overtorment** [comment]
@marcosrdz is it ready for merge? conflicts btw

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/bxqxefvydpww7nvcup25q0b1yc

[download apk](http://filestorage.bluewallet.io:1488/1581693144.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/80yq6zg6p35b41gaeb7v31yer8

[download apk](http://filestorage.bluewallet.io:1488/1581695157.apk)

File: screen/wallets/import.js [hunk 1]
```diff
@@ -108,9 +108,9 @@ const WalletsImport = () => {
           onPress={importButtonPressed}
         />
         <BlueButtonLink
-          title={loc.wallets.import.scan_qr}
+          title="...or scan for wallet instead?"
```
Comment (Overtorment) [inline screen/wallets/import.js]: why not `loc` ?

File: screen/wallets/transactions.js [hunk 2]
```diff
@@ -569,18 +578,48 @@ export default class WalletTransactions extends Component {
           })()}
 
           {(() => {
-            if (this.state.wallet.allowSend()) {
+            if (
+              this.state.wallet.allowSend() ||
+              (this.state.wallet._hdWalletInstance instanceof HDSegwitBech32Wallet && this.state.wallet._hdWalletInstance.allowSend())
```
Comment (Overtorment) [inline screen/wallets/transactions.js]: cant figure this out. elaborate?

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/b00ywt1120c99c7q69nb7hmne0

[download apk](http://filestorage.bluewallet.io:1488/1582581191.apk)

**Comment by Overtorment** [comment]
@marcosrdz can you fix 

```

    TypeError: Cannot read property 'RNFSFileTypeRegular' of undefined

      2 | import AsyncStorage from '@react-native-community/async-storage';
      3 | import BitcoinBIP70TransactionDecode from '../bip70/bip70';
    > 4 | import RNFS from 'react-native-fs';
```

in tests?

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/qczrurz1zchdryq65cmvkamymw

[download apk](http://filestorage.bluewallet.io:1488/1582634674.apk)

File: ios/BlueWallet/Info.plist [hunk 3]
```diff
@@ -58,18 +89,18 @@
...
 	<string>This alert should not show up as we do not require this data</string>
 	<key>NSCalendarsUsageDescription</key>
 	<string>This alert should not show up as we do not require this data</string>
 	<key>NSCameraUsageDescription</key>
-	<string>In order to quickly scan the recipient&apos;s address, we need your permission to use the camera to scan their QR Code.</string>
-	<key>NSLocationWhenInUseUsageDescription</key>
-	<string>This alert should not show up as we do not require this data</string>
+	<string>In order to quickly scan the recipient's address, we need your permission to use the camera to scan their QR Code.</string>
+	<key>NSFaceIDUsageDescription</key>
+	<string>In order to confirm your identity, we need your permission to use FaceID.</string>
```
[the remark this answers is not in the document]
Comment (ncoelho) [reply ios/BlueWallet/Info.plist]: <string>In order to use FaceID please confirm your permission</string>

File: screen/wallets/import.js (hunk 1 above)
Comment (ncoelho) [reply screen/wallets/import.js]: we can change the string to "scan QR or import file"

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/xytw2reahetwtynxnxnbaxuayc

[download apk](http://filestorage.bluewallet.io:1488/1582642167.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/57c5qbjbn247huztba6bkfrwe8

[download apk](http://filestorage.bluewallet.io:1488/1582728400.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/pyrrmpm2x8czg2vpmkm4a15bqr

[download apk](http://filestorage.bluewallet.io:1488/1582738857.apk)

**Pull request merged**